assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/BaseAssignmentService.java

```diff
@@ -10764,7 +10761,7 @@ public BaseAssignmentSubmission(String id, String assignId, String submitterId,
 			m_assignment = assignId;
 			m_properties = new BaseResourcePropertiesEdit();
 			addLiveProperties(m_properties);
-			m_submitters = new ArrayList();
+			m_submitters = new ArrayList<String>();
```

Comment:
Since 8, do you mean? I was thinking of starting to use lambdas in my code,
but wondered what the community would think about that.

On 21 March 2016 at 15:29, Matthew Buckett <notifications@github.com> wrote:

> In
> assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/BaseAssignmentService.java
> <https://github.com/sakaiproject/sakai/pull/1978#discussion_r56841729>:
>
> > @@ -10764,7 +10761,7 @@ public BaseAssignmentSubmission(String id, String assignId, String submitterId,
> >  			m_assignment = assignId;
> >  			m_properties = new BaseResourcePropertiesEdit();
> >  			addLiveProperties(m_properties);
> > -			m_submitters = new ArrayList();
> > +			m_submitters = new ArrayList<String>();
>
> You can always just the diamond operator: new ArrayList<>()
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/sakaiproject/sakai/pull/1978/files/25e5aeab7a885a525d273e565078dcdb331a5998#r56841729>
>